extensions/storage/README.md

```diff
@@ -0,0 +1,78 @@
+# Object Storage Extension Specification
+
+- **Title: Object Storage**
+- **Identifier: storage**
+- **Field Name Prefix: -**
+- **Scope: Item, Assets**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+- **Owner**: @davidraleigh
+
+This document explains the fields of the Object Storage Extension to a STAC Item and an Asset. This does not cover NFS solutions provided by PaaS cloud companies. Allows user to specify details related to cloud storage access and costs for assets and their STAC item.
+
+- [Example (Landsat 8)](examples/example-landsat8.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Asset fields
+
+| Field Name  | Type   | Description |
+| ----------- | ------ | ----------- |
+| storage:cloud_platform        | string    | The [cloud provider](#providers) where data is stored |
+| storage:manager               | string    | The entity in charge of managing the data. |
+| storage:regions               | [string]  | The region(s) where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
+| storage:bucket                | string    | The bucket for the asset, used along with object path |
+| storage:object_path           | string    | The object_path for the asset, used along with bucket |
+| storage:requester_pays        | bool      | Is the data requester pays or is it data manager/cloud provider pays |
+| storage:tier                  | string    | The title for the tier type (as defined by PaaS provider) |
+| storage:tier_duration         | integer   | Minimum storage duration required before additional fees |
+| storage:date_stored           | string    | Date and time the corresponding asset placed into the current storage tier (relevant for tier_duration > 0) |
+| storage:first_byte_latency    | string    | time unit (milliseconds, minutes or hours) for accessing first byte of data |
+
+## Item fields
+
+| Field Name  | Type   | Description |
+| ----------- | ------ | ----------- |
+| storage:min_tier_duration   | string  | Date and timestamp type describing the shortest time tier restrictions on access of assets. If `storage:archived_small` is false then thumbnails and other small overviews (implementer's discretion) are not included in `min_tier_duration` calculation |
+| storage:max_tier_duration   | string  | Date and timestamp type describing the longest time tier restrictions on access of assets. |
+| storage:archived_small         | bool      | Should the thumbnails or other small data be assumed to be included in the `min_tier_duration` and `max_tier_duration` |
```

Comment:
"small data" seems quite vague. If the `storage` extension is applied to each asset, this shouldn't be necessary, because you can just see if there are `storage` keys within the thumbnail asset?

Reply:
agree here, how do I know what is to be considered small?

Reply:
we can remove `archived_small` and just put the responsibility on the maintainer of the STAC service to not include thumbnails that are in Standard storage in the calculation of the min_tier_duration

Reply:
or to include them if they want all items archived

Reply:
https://github.com/radiantearth/stac-spec/pull/991/commits/09510ce3e91b73010a93991defdd51856c47c3b1